### PR TITLE
Respond with original file with attempting to resize animated gifs.

### PR DESCRIPTION
**Pull request by joehoyle**
When attempting to resize animated gifs we should send the original file backthrough as it's not currently supported.

### CODE REVIEW

File: lambda-handler.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
 var tachyon = require('./index');
 
+var proxyFile = require('./proxy-file')
 exports.handler = function(event, context, callback) {
 	var region = process.env.S3_REGION;
 	var bucket = process.env.S3_BUCKET;
 	var key = decodeURI(event.path.substring(1));
-	var args = event.queryStringParameters;
+	var args = event.queryStringParameters ? event.queryStringParameters : {};
```
Comment (rmccue) [inline lambda-handler.js]: The more idiomatic way would be `event.queryStringParameters || {}` I think

Comment (joehoyle) [inline lambda-handler.js]: 👍

File: lambda-handler.js [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
-	var args = event.queryStringParameters;
+	var args = event.queryStringParameters || {};
 	return tachyon.s3({ region: region, bucket: bucket }, key, args, function(
 		err,
 		data,
 		info
 	) {
 		if (err) {
+			if ( err.message === 'fallback-to-original' ) {
+				return proxyFile( region, bucket, key, callback )
```
Comment (rmccue) [inline lambda-handler.js]: `;`

File: lambda-handler.js [hunk 3]
```diff
@@ -1,16 +1,20 @@
 var tachyon = require('./index');
 
+var proxyFile = require('./proxy-file')
```
Comment (rmccue) [inline lambda-handler.js]: `;` and should probably be before the blank line, not after

File: proxy-file.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+				body: new Buffer(data.Body).toString('base64'),
+				isBase64Encoded: true,
+			};
+
+			callback(null, resp);
+		}
+	);
+}
+
+module.exports = sendOriginal
```
Comment (rmccue) [inline proxy-file.js]: `;`

**Pull request merged**